content/ch-gates/proving-universality.ipynb

```diff
@@ -370,7 +370,7 @@
     "qc.cx(0,1)\n",
     "qc.rx(theta,0)\n",
     "qc.cx(0,1)\n",
-    "qc.cx(0,2)\n",
+    "qc.cx(0,2)\n", 
```

Comment:
It seems like there is white space(?) added by accident? If so please remove it.

Reply:
Yes that's an accident, removed